Neos.Neos/Classes/NodeTypePostprocessor/CreationDialogPostprocessor.php

```diff
@@ -75,7 +75,7 @@ public function process(NodeType $nodeType, array &$configuration, array $option
         if (!isset($configuration['properties'])) {
             return;
         }
-        $creationDialogElements = [];
+        $creationDialogElements = $configuration['ui']['creationDialog']['elements'] ?? [];
```

Comment:
I think this will add an element twice if you only want to over-specify properties of the Creation Dialog (will check)

Reply:
No, it's all good – since it is merged in line 85